### DISCUSSION TITLE
add beta text in version 3 Payload CMS docs

### DISCUSSION THREAD

**Discussion by GerardPolloRebozado** [payloadcms/payload#9220] [Feature Requests & Ideas]
The docs for version 3 of payload used to show (beta) but this has changed.
I think it will be useful for people reading the docs to know that version 3 is a beta and its not stable

**Comment by tyteen4a03** [comment]
> The docs for version 3 of payload used to show (beta) but this has changed. I think it will be useful for people reading the docs to know that version 3 is a beta and its not stable

Stable is dropping next week!
